routes/teams.js

```diff
@@ -35,7 +35,7 @@ router.get('/quiz', function(req, res) {
 
   if (req.query.trending){
     if (type){ // leaders or goats
-      db.collection('teams').findOne( { _id : rb_team_id}, function (err, item){
+      db.collection('teams').findOne( { team_id : rb_team_id}, function (err, item){
```

Comment:
We weren't inserting a team_id field for leaders which is why it doesn't work from the league search page? I thought the idea was that the _id field in teams collection would then correspond to the team_id fields in the other collections?